### DISCUSSION TITLE
Change relationship picker to checkbox

### DISCUSSION THREAD

**Discussion by ducnhoxlatui** [Budibase/budibase#10715] [Help] [answered]
Hi guys,
I want to ask that how to change relationship picker to checkbox when creating a form.

Thanks

**Comment by ConorWebb96** [comment, marked as answer]
Hey @ducnhoxlatui,

Looking at the configuration of the field (screenshot below), there doesn't seem to be an option to change this from a multi-select to checkboxes. I'd recommend creating a potential enhancement request via issues.  https://github.com/Budibase/budibase/issues/new/choose
<img width="279" alt="Screenshot 2023-05-30 at 08 52 37" src="https://github.com/Budibase/budibase/assets/126772285/cbddc5a7-f759-490c-88ee-32d473ca2fa6">

Additionally, you could create a custom plugin and add this functionality there. https://docs.budibase.com/docs/custom-component Or you could fork the Budibase repo and add the functionality into the core product and then submit a PR for it to get merged in.

I hope this makes sense and answers your question.

**Reply by ducnhoxlatui** [reply]
So that mean there are no way to configure relationship picker to checkbox now in budibase, right? So the only way to do that is creating new component for that thing?

**Reply by ConorWebb96** [reply]
Yes, I believe the only component within Budibase that currently has the option for checkboxes is the `Multi-select picker`. The `options picker` has radio buttons instead of checkboxes.

**Reply by ducnhoxlatui** [reply]
yah, i know. So are there any solution to convert data from Multi-select picker to relationship picker? I try to use State but cannot.
